kitsune/products/jinja2/products/includes/topic_macros.html

```diff
@@ -1,9 +1,9 @@
-{% macro help_topics(topics, show_community_support=False) -%}
+{% macro help_topics(topics, show_community_support=False, product_slug=None) -%}
```

Comment:
Why are we changing this to accept `product_slug`s too?

Reply:
So we can use it in the aaq flow - in the aaq flow the slug isn't stored as `product.slug` as this macro assumes, but `product.product`, so we need to pass the slug explicitly rather than the whole "product" object.